src/wallet/api/wallet_manager.cpp

```diff
@@ -127,6 +127,8 @@ Wallet *WalletManagerImpl::createWalletFromDevice(const std::string &path,
     WalletImpl * wallet = new WalletImpl(nettype, kdf_rounds);
     if(restoreHeight > 0){
         wallet->setRefreshFromBlockHeight(restoreHeight);
+    } else {
+        wallet->setRefreshFromBlockHeight(wallet->estimateBlockChainHeight() - 1);
```

Comment:
I think the -1 should not be here. I saw simplewallet does this, I just changed it (unless you know of a good reason to do it).

Reply:
I did the same as in simplewallet, updated.